src/sentry/static/sentry/app/api.jsx

```diff
@@ -191,13 +191,12 @@ export class Client {
     return this.activeRequests[id];
   }
 
-  requestPromise(path, options = {}) {
+  requestPromise(path, {includeAllArgs, ...options} = {}) {
     return new Promise((resolve, reject) => {
       this.request(path, {
         ...options,
-        success: (data, ...args) => {
-          // This fails if we need jqXhr :(
-          resolve(data);
+        success: (data, _, allArgs) => {
```

Comment:
So we're ignoring the middle argument here `_`, let's make sure we resolve with exactly the same args so we avoid this problem again if something eventually needs this middle arg.